app/tests/integration/nimbus/conftest.py

```diff
@@ -320,6 +313,48 @@ def _language_database_id_loader(languages=None):
     return _language_database_id_loader
 
 
+@pytest.fixture(name="countries_database_id_loader")
+def fixture_countries_database_id_loader():
+    """Return database id's for languages"""
+
+    def _countries_database_id_loader(countries=None):
+        country_list = []
+        path = Path().resolve()
+        path = str(path)
+        path = path.strip("/tests/integration/nimbus")
+        path = os.path.join("/", path, "experimenter/base/fixtures/countries.json")
+        with open(path) as file:
+            data = json.loads(file.read())
+            for country in countries:
+                for item in data:
+                    if country in item["fields"]["code"][:2]:
+                        country_list.append(item["pk"])
+        return country_list
+
+    return _countries_database_id_loader
+
+
+@pytest.fixture(name="locales_database_id_loader")
+def fixture_locales_database_id_loader():
+    """Return database id's for languages"""
+
+    def _locales_database_id_loader(locales=None):
+        locale_list = []
+        path = Path().resolve()
+        path = str(path)
+        path = path.strip("/tests/integration/nimbus")
+        path = os.path.join("/", path, "experimenter/base/fixtures/locales.json")
+        with open(path) as file:
+            data = json.loads(file.read())
+            for locale in locales:
+                for item in data:
+                    if locale in item["fields"]["code"]:
+                        locale_list.append(item["pk"])
+        return locale_list
+
+    return _locales_database_id_loader
```

Comment:
I could probably write one function to do all of this, but maybe I save that for a contributor??

Reply:
Actually the other thing I was thinking is you could also query this info from the configuration endpoint.  I thought we were doing that at some point but now I don't see it.  Or did we figure out that it was too slow/janky?  Reading them from the disk works too.

Reply:
I'd need help with that if you think it's worth doing.